packages/sourcecred/src/plugins/discord/config.js

```diff
@@ -66,32 +66,31 @@ export type DiscordConfigJson = {|
   +propsChannels?: $ReadOnlyArray<Model.Snowflake>,
   // Whether to include NSFW channels in cred distribution or not
   +includeNsfwChannels?: boolean,
-  // What the weight should be for reactions not specified in reactionWeights
-  +defaultReactionWeight?: number,
-  // Whether reaction weights on a given message should be divided by the number
-  // of unique members that reacted to the message.
-  +applyAveragingToReactions?: boolean,
-|};
+|}>;
 
-const parserJson: C.Parser<DiscordConfigJson> = C.object(
-  {
-    guildId: C.string,
-    reactionWeights: C.dict(C.number),
```

Comment:
I just wanted to make sure that we're marking that obsolete, I think that's a good idea for a breaking to change to like put people into perspective, as the current template instance would have to be changed to adopt `reactionWeightConfig` instead

Reply:
See: https://github.com/sourcecred/sourcecred/pull/3065

Reply:
Awesome